### ISSUE TITLE
🐛  Error: Should have returned a node

### ISSUE THREAD

**Issue by rickhanlonii** [rome/tools#992]
## Environment

<details><pre>Environment Variables
  Object {
    TERM: "xterm-256color"
    USER: "rickhanlonii"
  }

User Config
  unset

Rome Version
  10.0.3-beta

Node Version
  12.16.1

Platform
  darwin x64 19.4.0

Terminal Features
  Object {
    background: "unknown"
    colorDepth: 8
    columns: 231
    cursor: true
    hyperlinks: true
    isTTY: true
    unicode: true
  }

Client Flags
  Object {
    clientName: "cli"
    cwd: "/Users/rickhanlonii/dev/oss/react"
    realCwd: "/Users/rickhanlonii/dev/oss/react"
    silent: false
  }

Server Status
  Object {
    projects: Array [
      Object {id: 0}
      Object {id: 1}
    ]
    server: Object {
      heapTotal: 26_316_800
      pid: 42_498
      uptime: 0.237943279
    }
    workers: Array [
      Object {
        astCacheSize: 0
        heapTotal: 26_316_800
        ownedBytes: 0
        ownedFileCount: 0
        pid: 42_498
        uptime: 0.237854373
      }
    ]
  }
</pre></details>

## Steps To Reproduce

1. clone facebook/react
2. npx rome init
3. npx rome check

## Expected Results

Either the result of the check or an error explaining what is configured wrong.

## Actual Result

<img width="1637" alt="Screen Shot 2020-08-08 at 5 58 13 PM" src="https://user-images.githubusercontent.com/2440089/89720621-0466d980-d9a2-11ea-904b-725644471be0.png">

**Comment by ematipico** [comment]
Quick question about the file that rome tried to lint (based on the screenshot): is that file a flow file?

**Comment by rickhanlonii** [comment]
[Here's the file](https://github.com/facebook/react/blob/master/scripts/jest/matchers/interactionTracingMatchers.js), I don't see anything flow specific in the file.

**Comment by ematipico** [comment]
The file should be `react-native-host-hooks.js`.

It's under the stack trace

**Comment by sebmck** [comment]
Yeah it's a Flow file but still a bug since the parser should never throw runtime errors.

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
